templates/data/prow-periodics-data.yaml

```
@@ -10,23 +10,13 @@ templates:
             type_periodic: "true"
             cluster: "trusted-workload"
           github_token_mounts:
-            volumes:
-              - name: kyma-bot-github-token
-                secretName: kyma-bot-github-token
-            volumeMounts:
-              - name: kyma-bot-github-token
-                mountPath: /etc/github
-                readOnly: true
+            labels:
+              kyma-autobump-bot-github-token: "true"
```

Preset label name is different.
```suggestion
              preset-autobump-bot-github-token: "true"
```